bucket/syncbackfree.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "9.2.12.0",
+    "description": "Data backup and synchronize software",
+    "homepage": "https://www.2brightsparks.com/download-syncbackfree.html",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.2brightsparks.com/download-syncbackfree.html#tab2"
+    },
+    "url": "https://www.2brightsparks.com/assets/software/SyncBack_Setup.exe",
+    "hash": "4995b24b6a9c2b3e71365a967b469220bfbe4f4bd42ddd4133714f1938a0a6f8",
```

Comment:
Latest version in download page is `9.2.30.0` 
Hash in this manifest is for version 9.2.12.0, but this hash is same for 30

![image](https://user-images.githubusercontent.com/13260377/72224359-0686c800-357a-11ea-9330-06f877e70551.png)